static/js/markdown.js

```diff
@@ -26,11 +26,11 @@ let helpers;
 
 // Regexes that match some of our common backend-only Markdown syntax
 const backend_only_markdown_re = [
-    // Inline image previews, check for contiguous chars ending in image suffix
+    // Inline image and video previews, check for contiguous chars ending in image and video suffix
     // To keep the below regexes simple, split them out for the end-of-message case
 
-    /\S*(?:\.bmp|\.gif|\.jpg|\.jpeg|\.png|\.webp)\)?\s+/m,
-    /\S*(?:\.bmp|\.gif|\.jpg|\.jpeg|\.png|\.webp)\)?$/m,
+    /\S*(?:\.bmp|\.gif|\.jpg|\.jpeg|\.png|\.webp|\.mp4|\.webm|\.mov|\.avi|\.wmv)\)?\s+/m,
+    /\S*(?:\.bmp|\.gif|\.jpg|\.jpeg|\.png|\.webp|\.mp4|\.webm|\.mov|\.avi|\.wmv)\)?$/m,
```

Comment:
Tangential, but there's no reason to make this two regexes:
```
/\S*(?:\.bmp|\.gif|\.jpg|\.jpeg|\.png|\.webp|\.mp4|\.webm|\.mov|\.avi|\.wmv)\)?(\s+|$)/m,
```

Reply:
Sure, added another commit for this.